app.js

```diff
@@ -31,9 +31,36 @@
 // 74 - Soft Tek Front
 
 let drums = null;
+let buttons = null;
```

Comment:
I think this should be initialized after `DEFAULT_BUTTONS` and set as a copy of it.